java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -95,4 +95,95 @@ public interface BitmapBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> setbit(String key, long offset, long value);
+
+    /**
+     * Return the position of the first bit matching the given <code>bit</code> value.
+     *
+     * @see <a href="https://redis.io/commands/bitpos/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param bit The bit value to match.
+     * @return The position of the first occurrence matching <code>bit</code> in the binary value of
+     *     the string held at <code>key</code>. If <code>bit</code> is not found, a <code>-1</code> is
+     *     returned.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.bitpos("myKey1", 0).get();
+     * assert payload == 3L; // Indicates that the first occurrence of a 0 bit value of the string stored at "myKey1" is at the fourth position.
+     * }</pre>
+     */
+    CompletableFuture<Long> bitpos(String key, long bit);
+
+    /**
+     * Return the position of the first bit matching the given <code>bit</code> value. The offset
+     * <code>start</code> is a zero-based index, with <code>0</code> being the first byte of the list,
+     * <code>1</code> being the next byte and so on. These offsets can also be negative numbers
+     * indicating offsets starting at the end of the list, with <code>-1</code> being the last byte of
+     * the list, <code>-2</code> being the penultimate, and so on.
+     *
+     * @see <a href="https://redis.io/commands/bitpos/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param bit The bit value to match.
+     * @param start The starting offset.
+     * @return The position of the first occurrence beginning at the <code>start</code> offset of the
+     *     <code>bit</code> in the binary value of the string held at <code>key</code>.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.bitpos("myKey1", 1, 4).get();
+     * assert payload == 9L; // Indicates that the first occurrence of a 1 bit value starting from fifth byte of the string stored at "myKey1" is at the tenth position.
```

Comment:
The fifth byte starts at bit position 32 so the result will have to be >= 32
```suggestion
     * assert payload == 33L; // Indicates that the first occurrence of a 1 bit value starting from fifth byte of the string stored at "myKey1" is at position 33.
```